src/fs/mod.rs

```diff
@@ -0,0 +1,34 @@
+//! A high-level file system API for UEFI applications. It supports you to conveniently perform the
+//! following important operations:
+//! - read file to bytes
+//! - write bytes to file
+//! - create files
+//! - iterate directories and walk the file system tree
+//!
+//! Unlike UNIX-based file systems, there is no virtual file system. Thus, users perform actions on
```

Comment:
Instead of just saying that this is not like UNIX, we could also affirmatively indicate that it's a look like Windows, where you also have distinct "drive letters" for each volume :)